src/Shape/Pie.elm

```diff
@@ -1,4 +1,4 @@
-module Shape.Pie exposing (..)
+module Shape.Pie exposing (arc, arc_, boolToArc, boolToDirection, centroid, cornerTangents, epsilon, intersect, makeArc, mod, myAsin, pie)
```

Comment:
This is almost certainly not required. I fairly confident that the only exports required here are `centroid` and `pie`.

Reply:
Agreed, that seems to be the default bahavior of elm-format if you expose everything. AFAICT only `arc` and `centroid` are used within the codebase. Happy to change `exposing` to only include those two funtcionts, but it'll require bumping the library version since the interface changes. What do you think?

Reply:
That module isn't exposed, so it will be ok.

Reply:
Done, thanks!